src/Models/LagrangianParticleTracking/lagrangian_particle_advection.jl

```diff
@@ -78,7 +78,7 @@ Return new position `(x⁺, y⁺, z⁺)` for a particle at current position (x,
 given `velocities`, time-step `Δt, and coefficient of `restitution`.
 """
 @inline function advect_particle((x, y, z), p, restitution, grid, Δt, velocities)
-    X = flattened_node((x, y, z), grid)
+    X = @show flattened_node((x, y, z), grid)
```

Comment:
probably want to delete this before merging?

Reply:
Good catch thank you